lms/lmsweb/views.py

```diff
@@ -40,6 +42,18 @@
 HIGH_ROLES = {str(RoleOptions.STAFF), str(RoleOptions.ADMINISTRATOR)}
 MAX_REQUEST_SIZE = 2_000_000  # 2MB (in bytes)
 
+DIRECTION = 'rtl' if LOCALE == 'he' else 'ltr'
```

Comment:
We should have somewhere a `set()` of rtl languages and `DIRECTION` should be set to `rtl` if `LOCATE in RTL_LANGUAGES`